README.md

```diff
@@ -20,3 +20,45 @@ limitations under the License.
 <b>[Apache Phoenix](http://phoenix.apache.org/)</b> enables OLTP and operational analytics in Hadoop for low latency applications. Visit the Apache Phoenix website <b>[here](http://phoenix.apache.org/)</b>. This repo contains connectors for third party libraries to access data stored in Phoenix/HBase. 
 
 Copyright ©2019 [Apache Software Foundation](http://www.apache.org/). All Rights Reserved. 
+
+## Introduction
+This repo contains Flume, Pig, Kafka, Spark and Hive connectors for Phoenix 4 and for Phoenix 5.
+
+
+## Building
+
+This repository will build jars for the different phoenix connectors.
+Connectors for Phoenix 4 use Hadoop 2 and HBase 4 meanwhile Phoenix 5 provides compatibility for Hadoop 3 and Hbase 5.
+All Phoenix 4 and Phoenix 5 connectros are built in succession.
```

Comment:
I'd prefer:
The standard build process builds the connectors both for Phoenix 4 and Phoenix 5.